### ISSUE TITLE
How can I reduce the size of the path?

### ISSUE THREAD

**Issue by pbragamiranda** [zthxxx/jovial#30]
Hi! Thanks for the amazing work, I'm loving jovial. One thing that I wish I could customize is the size of the path.

right now I have the full path `~/code/manegarrincha/portfolio/app/assets/stylesheets` 

I wish I could have just `~/stylesheets` . 

Or perhaps choose the number of folders displaying. i.e `folders{2}` -> `~/assets/stylesheets` 

I've tried to look into but wasn't successful =/

**Comment by zthxxx** [comment]
@pbragamiranda  Sorry for a bug in v2.5.0 that causes responsive style error due to the new `path` part,

I have fixed in v2.5.1 which maybe you would to upgrade to it, then rename the `path` part to `current-dir` in your `.zshrc`

```diff
- JOVIAL_AFFIXES[path]='%2~'
+ JOVIAL_AFFIXES[current-dir]='%2~'
```

**Comment by pbragamiranda** [comment]
Great! Thanks for letting me know 🙃
